### PR TITLE
Fixed issue with TestNG not getting garbage collected. See #1461

### PR DESCRIPTION
**Pull request by kiru**
This change should allow JVM to GC the TestNG instance after tests are completed.

### CODE REVIEW

**Comment by kiru** [comment]
You mean, like a normal test?

   ```
   @Test
    public void memoryLeakTest() throws InterruptedException {
        // we run the test programmatically
        runTest();
        // lets wait for garbage collection
        waitForAllObjectsDestructed();
    }
```
The problem is that I creat a TestNG instance within a test and wait that intance to get GCed, but that wont happen.
That is the reason to write it with the main() method. Do you know another solution?

**Comment by juherr** [comment]
Most of the testng tests are creating a testng instance. That's why I asking if your test can work with this common pattern too. ;)

**Comment by kiru** [comment]
Yeah, but I have no other idea how to write a TestNG test and run it with TestNG to check if TestNG instnace get garbage collected.

File: src/test/java/test/github1471/TestNgEnableBehaviour.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test test should ensure that disabled tests are not executed.
+ * See issue: https://github.com/cbeust/testng/issues/1471
+ *
+ * @author Kirusanth Poopalasingam ( code@kiru.io )
+ */
+public class TestNgEnableBehaviour extends SimpleBaseTest{
```
Comment (juherr) [inline src/test/java/test/github1471/TestNgEnableBehaviour.java]: This test class has no relation to the issue. Could you remove the commit from the branch?

**Pull request closed without merging**